src/libs/antares/study/importlogs.cpp

```diff
@@ -36,9 +36,9 @@ void Study::importLogsToOutputFolder(Solver::IResultWriter& resultWriter) const
     if (!logs.logfile())
         return;
 
-    std::string logPath("simulation.log");
-    String from;
-    IO::Normalize(from, logs.logfile());
+    std::filesystem::path logPath = "simulation.log";
```

Comment:
```C++
std::filesystem::path logPath = "simulation.log";
```
is not a log path, it's rather a filename. 
**logPath**  == > **logFileName** ?

Reply:
And I'am not even sure that we need **logPath** : 
```C++
resultWriter.addEntryFromFile("simulation.log", from.string());
```
should work